src/extra_files.c

```diff
@@ -96,20 +96,21 @@ static bool handle(const char *filename, bool is_dir, bool fix)
 	char *temp;
 	bool ret = true;
 
+	remove_trailing_slash(globals.path_prefix);
```

Comment:
yeah, I didn't think it was a big deal because this function is being called basically by the last step of verify, so even if the global variable changes it would not affect anything else. But you are right it is better to use sys_path_join() instead,